src/ToggleBadge.js

```diff
@@ -12,7 +12,7 @@ const ToggleBadge = styled.button`
   font-family: inherit;
   cursor: pointer;
   background-color: ${props =>
-    props.selected ? props.theme.colors[props.bg] : props.theme.colors.white};
+    props.selected ? props.theme.colors[props.bg] : 'transparent'};
```

Comment:
Can you check on the design-system channel to see if anyone else is using this component, and if so, would a transparent background break their design?  This could be considered a breaking change under certain use cases.

Reply:
thats a good point.

Reply:
though I doubt its being used anywhere else

Reply:
Think another option is to make this "_transparent_" style optional, controlled by a prop name, namely 'isTransparent` perhaps? What do you guys think, @surabhinigam & @jes708?

Reply:
good point @BeniCheni , I have addressed the issue by taking background color for an unselected badge as prop, and keeping `transparent` as default. How does that sound to you?